ddln_imaginator.proto

```diff
@@ -185,7 +186,45 @@ message WorldInfo {
     optional LatLongAlt home_geo = 4;
 }
 
+// Imaginator info message which contains imaginator specific information
+// This information is world agnostic and typically generated upon initialization
+message ImaginatorInfo {
+
+    // Reserved for future values such as version info and so on.
+    reserved 1, 2, 3, 4, 5, 6, 7, 8, 9;
+
+    // Object information message
+    message ObjectInfo {
+
+        // True if object info is customized, false in case information is left undefined, e.g. not specified.
+        // Contact sim@daedalean.ai if object information for a specific object key is required.
+        optional bool has_info = 1;
+
+        // Object full name
+        optional string name = 2;
+
+        // Optional short description of the object. Often left empty.
+        optional string description = 3;
+
+        // Annotation class as per Daedalean spotter ground truth annotation requirements
+        enum Class {
+            UNDEFINED = 0;
+            DRONE = 1;
+            BIRD = 3;
+            FIXED_WING = 4;
+            HELICOPTER = 5;
+            UNKNOWN = 100;
```

Comment:
`UNKNOWN` => `OTHER`
otherwise there's confusion between `UNKNOWN` and `UNDEFINED`

Reply:
UNKNOWN is per Daedalean annotation requirements.